docs/running-locally.md

```diff
@@ -206,6 +221,34 @@ Tests often fail: that's good. To diagnose failure:
 
 If tests run slowly or time out, factory reset your Kubernetes cluster.
 
+### Debugging using Visual Studio Code
+
+When using the Dev Container with VSCode, add the following launch configuration to `.vscode/launch.json`:
```

Comment:
We could just add this to `.vscode/launch.json` directly instead of having a snippet here. Especially as we already have devcontainer config and Nix config, so there is precent for that. I had been thinking of doing that myself as well at some point

Reply:
There were two reasons I didn't directly commit `.vscode/launch.json`:
1. Anyone that already has a `.vscode/launch.json` will have it overwritten by Git the next time they do a `git pull`
2. It's common to add ad-hoc launch configurations for debugging specific things, which we don't want to commit

But I found an alternative that should avoid both problems: specifying default launch configurations inside `devcontainers.json`. I went ahead and did that in c430b3c7e0a3a6159270d939f945c06ea1c7594a. Docs: https://containers.dev/supporting#visual-studio

Reply:
> 2\. It's common to add ad-hoc launch configurations for debugging specific things, which we don't want to commit

I don't necessarily disagree with this and I have done this before. Though if they're temporary, I just don't commit those changes. Or sometimes I chuck them into my user config instead.

> But I found an alternative that should avoid both problems:

This seems clean enough to me though